src/index.html

```diff
@@ -7,6 +7,41 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <!-- <h1>Calendar</h1> -->
+    <main class="container">
+      <section class="calendar calendar--start-day--fri calendar--month-length--29">
+        <div class="calendar__day-box"></div>
+        <div class="calendar__day-box"></div>
+        <div class="calendar__day-box"></div>
```

Comment:
I would say just day is enough here
```suggestion
        <div class="calendar__day"></div>
        <div class="calendar__day"></div>
        <div class="calendar__day"></div>
```